### PR TITLE
Remove unused code about `sider.yml` for `metrics_*` runners

### PR DESCRIPTION
**Pull request by ybiquitous**
> Explain a summary, purpose, or background for this change.

Metrics runners now do not use `sider.yml`.

> Link related issues or pull requests.

None.

> Check the following if needed.

- [ ] Add a new entry to [CHANGELOG.md](https://github.com/sider/runners/blob/master/CHANGELOG.md) if this change is notable.

### CODE REVIEW

**Review by yykamei — APPROVED** [review]
LGTM

File: test/smokes/metrics_complexity/expectations.rb [hunk 1]
```diff
@@ -142,58 +142,6 @@
   analyzer: { name: "Metrics Complexity", version: default_version }
 )
 
-s.add_test(
-  "root_dir",
-  type: "success",
-  issues: [
-    {
-      id: "metrics_file-complexity",
-      path: "src/baz/fizzbuzz.py",
```
Comment (suetsuguk) [inline test/smokes/metrics_complexity/expectations.rb:151]: Test cases for .py, .go and .cs go missing from the entire test.
Can you restore followings and put them into the existing test named "success"?
- src/baz/fizzbuzz.py
- src/baz/qux/今日は世界.go
- src/foo/こんにちは世界.cs

Comment (ybiquitous) [reply test/smokes/metrics_complexity/expectations.rb:151]: Done. 0dc12e49

**Review by suetsuguk — APPROVED** [review]
LGTM. Thank you.

**Comment by ybiquitous** [comment]
Thanks all for the review. 🥰

**Pull request merged**